packages/connectors/connector-metamask/src/types.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from 'zod';
+
+export const metaMaskConfigGuard = z.object({});
```

Comment:
📝 Look into what config can be provided. Maybe users want to limit the chains available? 🤷